static/app/components/gridEditable/index.tsx

```diff
@@ -95,7 +95,7 @@ type GridEditableProps<DataRow, ColumnKey> = {
   };
   location: Location;
   emptyMessage?: React.ReactNode;
-  error?: React.ReactNode | null;
```

Comment:
in this case it was never a ReactNode its an Error object of some kind. We just check if it exists so unknown is okay to check truthy